### PR TITLE
LUCENE-10236: Add change entry

### PR DESCRIPTION
**Pull request by zacharymorn**

### CODE REVIEW

File: lucene/CHANGES.txt [hunk 1]
```diff
@@ -57,7 +57,9 @@ Optimizations
 
 Bug Fixes
 ---------------------
-(No changes)
+
+* LUCENE-10236: Update field-weight used in CombinedFieldQuery scoring calculation.
```
Comment (mikemccand) [inline lucene/CHANGES.txt:61]: Maybe say a bit more about what really changed than just "update", e.g. `stop duplicating norms and term frequencies when scoring in CombinedFieldQuery`?

Comment (zacharymorn) [reply lucene/CHANGES.txt]: Ah sorry. I've updated it accordingly.

Comment (zacharymorn) [reply lucene/CHANGES.txt]: Hi @mikemccand , just want to check back on this to see if the updated entry looks good to you?

Comment (mikemccand) [reply lucene/CHANGES.txt]: Oh yes looks great sorry for the slow response!

**Review by mikemccand — APPROVED** [review]
Thanks @zacharymorn

**Comment by mikemccand** [comment]
Hmm actually this issue was fixed in 9.0.0, I think?

File: lucene/CHANGES.txt (hunk 1 above)
Comment (zacharymorn) [reply lucene/CHANGES.txt]: No problem, thanks @mikemccand for the review and approval!

**Comment by zacharymorn** [comment]
> Hmm actually this issue was fixed in 9.0.0, I think?

This change entry is for https://github.com/apache/lucene/commit/07ee3ba83a4c9f3abc24bf9d3fbb3c3102c4a102#diff-27a1984493254d0af3e9c50caee47443abbbdc79a87b4c348300bcb317a0a5b2, as I forgot to add it in that merge. I checked the commit immediately after mine https://github.com/apache/lucene/commit/ad911df2605591a63ca58d237182f84aa8384325# on `main` and saw the change entry used `9.1.0`, so I assumed I should use that as well?

On the other hand, [`branch_9_0`](https://github.com/apache/lucene/commits/branch_9_0) doesn't contain my fix commit as I haven't back-ported it yet.

Not sure if I may miss something?

**Comment by zacharymorn** [comment]
> > Hmm actually this issue was fixed in 9.0.0, I think?
> 
> This change entry is for [07ee3ba#diff-27a1984493254d0af3e9c50caee47443abbbdc79a87b4c348300bcb317a0a5b2](https://github.com/apache/lucene/commit/07ee3ba83a4c9f3abc24bf9d3fbb3c3102c4a102#diff-27a1984493254d0af3e9c50caee47443abbbdc79a87b4c348300bcb317a0a5b2), as I forgot to add it in that merge. I checked the commit immediately after mine [ad911df](https://github.com/apache/lucene/commit/ad911df2605591a63ca58d237182f84aa8384325) on `main` and saw the change entry used `9.1.0`, so I assumed I should use that as well?
> 
> On the other hand, [`branch_9_0`](https://github.com/apache/lucene/commits/branch_9_0) doesn't contain my fix commit as I haven't back-ported it yet.
> 
> Not sure if I may miss something?

Hi @mikemccand , just want to check back on this and see if you have any further concern?

**Comment by mikemccand** [comment]
Aha, perfect, looks great!  +1 to push.  Sorry for the delay @zacharymorn.

**Review by mikemccand — APPROVED** [review]

**Comment by zacharymorn** [comment]
> Aha, perfect, looks great! +1 to push. Sorry for the delay @zacharymorn.

No problem! Thanks for the review and approval!

**Pull request merged**